website/content/currencies.md

```diff
@@ -12,7 +12,8 @@ weight: 230
 | AFN  | 971 | 2   |                              |
 | ALL  | 008 | 2   |                              |
 | AMD  | 051 | 2   |                              |
-| ANG  | 532 | 2   |                              |
+| ANG  | 532 | 2   | Not recognized by Visa       |
+| XCG  | 532 | 2   | Not recognized by Mastercard |
```

Comment:
Wouldn't it make sense to keep the table sorted on the alpha code?

Reply:
I wanted it to be easy to see that they belong together so I purposely ignored the sorting for this entry. I think it makes a bit more sense to keep them together. This is the only duplicate code so there is no precedence.